web-registry/src/components/templates/CreditBatchFormTemplate.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
I think that would make sense to turn this into a more generic form template and take the steps as props

Reply:
I think I have a better implementation now. This file is deleted, and the parent component is called `CreditBatchIssuance`